### PR TITLE
Add walk-and-graph-svelte-components

### PR DESCRIPTION
**Pull request by j2l**
Just adding my tool for the great society :)

### CODE REVIEW

File: src/routes/tools/tools.json [hunk 1]
```diff
@@ -322,5 +322,15 @@
 		"addedOn": "2021-12-08",
 		"tags": [],
 		"stars": 0
-	}
+	},
+	{
+		"title": "walk-and-graph-svelte-components",
+		"url": "https://github.com/j2l/walk-and-graph-svelte-components",
+		"description": "CLI node script to walk svelte and js files, to draw a beautiful JPG of your dependencies aka \"imports\". No external dependency, only FS (Node). ",
+		"npm": "none",
```
Comment (MacFJA) [inline src/routes/tools/tools.json]: Instead of `"npm": "none",` you can completely remove this line

**Review by MacFJA — CHANGES_REQUESTED** [review]

File: src/routes/tools/tools.json (hunk 1 above)
Comment (j2l) [reply src/routes/tools/tools.json]: Ok

**Review by j2l — COMMENTED** [review]
I changed it but don't know how to approve this.

File: src/routes/tools/tools.json [hunk 2]
```diff
@@ -322,5 +322,14 @@
...
+	},
+	{
+		"title": "walk-and-graph-svelte-components",
+		"url": "https://github.com/j2l/walk-and-graph-svelte-components",
+		"description": "CLI node script to walk svelte and js files, to draw a beautiful JPG of your dependencies aka \"imports\". No external dependency, only FS (Node). ",
+		"addedOn": "2022-02-03",
+		"category": "Debugging",
+		"tags": [],
+		"stars": 0
+}
```
Comment (MacFJA) [inline src/routes/tools/tools.json]: This line is not well formatted.

You can run `npm run format` to fix it automatically.
And `npm run lint` yo check if everything is ok

**Review by MacFJA — CHANGES_REQUESTED** [review]

File: src/routes/tools/tools.json (hunk 2 above)
Comment (j2l) [reply src/routes/tools/tools.json]: Which one?
BTW, it's generated by your tool so I leave you fixing it, thanks.

**Comment by benmccann** [comment]
this PR would need to be rebased

**Pull request merged**